test/e2e/providers/remote.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2019 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package providers
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"os"
+	"path"
+
+	"github.com/kevinburke/ssh_config"
+
+	"github.com/vmware-tanzu/antrea/test/e2e/providers/exec"
+)
+
+var (
+	homedir, _       = os.UserHomeDir()
+	sshConfig        = flag.String("remote.sshconfig", path.Join(homedir, ".ssh", "config"), "Path of the sshconfig")
+	remoteKubeconfig = flag.String("remote.kubeconfig", path.Join(homedir, ".kube", "config"), "Path of the kubeconfig of the cluster in local")
```

Comment:
I'm trying to tell user the path to the kubeconfig is in local machine but not the remote. But looks like the conventional practice would always expect a local path, removing it would be better.